orchestrator-salt/src/main/resources/salt-common/salt/fluent/cron/cdp_logging_agent_doctor.sh

```diff
@@ -0,0 +1,2 @@
+# run logging agent doctor every day at 01:00:00
+0 1 * * * root sh /opt/salt/scripts/cdp_logging_agent_check.sh doctor
```

Comment:
yes, this script will nothing to do with node status monitor, although would make sense to support the same mechanism on cdp-doctor side....here the point is to update this script (e.g. on diagnostics execution) on any CB code changes, but cdp-doctor related binaries are installed already on the nodes (which can be much older or newer etc.), so that's more like an individual thing with similar concept (continously run some check commands as cdp-nodestatus-monitor does with cdp-doctor)....maybe if a mechanism like this can be a part of the cdp-telemetry binary, that crontab can be the part of the rpm, so there can be.a condition here later to not create this script if that's exist...this can be added later i think